tensorrt_llm/_torch/modules/rms_norm.py

```
@@ -132,9 +136,22 @@ def _ensure_contiguous_with_dtype(t: torch.Tensor, key: str):
+                return (hidden_states_fused,
+                        residual_out) if has_residual else hidden_states_fused
+            else:
+                high_precision_normed_output = results[3].reshape(orig_shape)
+                return (hidden_states_fused, residual_out,
+                        high_precision_normed_output) if has_residual else (
+                            hidden_states_fused, high_precision_normed_output)
+
+        if self.return_hp_output:
+            raise ValueError(
+                "Auxiliary high precision output is only supported for NVFP4 fused path"
+            )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)